app/assets/stylesheets/admin.scss

```diff
@@ -32,9 +31,34 @@
   }
 }
 
+// Weekly stats
+.week-selector {
+  margin-top: 18px;
+}
+
+// Pod stats
+.segment-selection {
+  select,
+  [type=submit] {
+    margin-top: 10px;
```

Comment:
But `margin-top` is set to 18px here https://github.com/diaspora/diaspora/pull/7295/files#diff-111bd41e9b25b9eaafc015184274028dR40